app/actions/remote/search.ts

```diff
@@ -138,14 +138,9 @@ export const searchFiles = async (serverUrl: string, teamId: string, params: Fil
     }
 
     let client: Client;
-    try {
-        client = NetworkManager.getClient(serverUrl);
-    } catch (error) {
-        return {error};
-    }
-
     let data;
     try {
+        client = NetworkManager.getClient(serverUrl);
```

Comment:
you can remove the let in line 140 and do
```suggestion
        const client = NetworkManager.getClient(serverUrl);
```

Reply:
in fact you can also remove everything above the try as you don't need the operator in this function